test/com/googlecode/jmxtrans/model/output/CloudWatchWriterTests.java

```diff
@@ -0,0 +1,30 @@
+package com.googlecode.jmxtrans.model.output;
+
+import com.googlecode.jmxtrans.model.Query;
+import com.googlecode.jmxtrans.model.Server;
+import com.googlecode.jmxtrans.model.ValidationException;
+import org.junit.Test;
+
+/**
+ * Tests for {@link com.googlecode.jmxtrans.model.output.CloudWatchWriter}.
+ *
+ * @author Sascha Moellering <moelleri@amazon.de>
+ */
+public class CloudWatchWriterTests {
```

Comment:
Hi,

this looks good! Yesterday I spent some time to implement the CloudWatch part to jmxtrans-agent, I'll open a PR next week for that.